### ISSUE TITLE
--cache-from and --cache-to

### ISSUE THREAD

**Issue by cierzniak** [thegeeklab/drone-docker-buildx#47]
How to use `--cache-from` and `--cache-to` flags? ([docs](https://docs.docker.com/engine/reference/commandline/buildx_build/#cache-from)).

For mine Raspberry Pi which builds arm64 and amd64 images with nodejs it's very time ineffective to build same node_modules each time or even retries pipeline after failuers.

**Comment by cierzniak** [comment]
```
latest: Pulling from thegeeklab/drone-docker-buildx
Digest: sha256:4c341cc426bf473db7de3ae0bc9a8479c92fbdfde9f67c49893771590065a447
Status: Image is up to date for thegeeklab/drone-docker-buildx:latest
+ /usr/local/bin/dockerd --data-root /var/lib/docker --host=unix:///var/run/docker.sock
Detected registry credentials

...

+ /usr/local/bin/docker pull registry.******.it/app/muzeum:cache
cache: Pulling from app/muzeum
no matching manifest for linux/arm64/v8 in the manifest list entries
Could not pull cache-from image registry.******.it/app/muzeum:cache. Ignoring...
+ /usr/local/bin/docker buildx build --rm=true -f Dockerfile . --push --pull=true --cache-from registry.******.it/app/muzeum:cache --build-arg DOCKER_IMAGE_CREATED=2021-08-29T22:06:48Z --platform linux/arm64,linux/amd64 -t registry.******.it/app/muzeum:latest

...

importing cache manifest from registry.******.it/app/muzeum:cache
DONE 0.3s
```

so it's working or not? on local mac with m1 (docker desktop) after prune

command :
```
docker buildx build -t registry.******.it/app/muzeum:latest --cache-from=registry.******.it/app/muzeum:cache --cache-to=registry.******.it/app/muzeum:cache  --platform linux/arm64,linux/amd64 . --push
```

![image](https://user-images.githubusercontent.com/815618/131341915-36bf95ff-0ee5-495f-9071-6dfca6655d78.png)

and loading everything from cache.

---

screen for proof that cache tag is available in registry:
![image](https://user-images.githubusercontent.com/815618/131342244-8bcc2671-94bd-4d58-9e82-5ce33a8431b1.png)

**Comment by xoxys** [comment]
As I said, the `--cache-to` flag is currently not available in this Drone plugin and the `cache_from` plugin parameter works exactly the same as the `--cache-from` cli flag, see https://github.com/thegeeklab/drone-docker-buildx/blob/main/plugin/docker.go#L103

As a workaround for the `cache_to` behavior, you could use drone-docker-buildx in the same way you would use it to publish your final image to a registry but this time you use e.g. the commit hash or `cache` as a tag.

**Comment by xoxys** [comment]
Every further docker build step in a CI run could use `cache_from: "registry.example.com/myorg/myapp:cache"`.

**Comment by cierzniak** [comment]
Ok, thanks for help :)

**Comment by djmaze** [comment]
The proposed workaround does not work for me. So I implemented the `cache_to` option for real in #124.
